llvm/lib/MCCAS/MCCASObjectV1.cpp

```diff
@@ -1893,7 +1893,9 @@ struct DIEDataWriter : public DataWriter {
 
   /// Saves the main data stream and any children to a new DIEDataRef node.
   Expected<DIEDataRef> getCASNode(MCCASBuilder &CASBuilder) {
-    return DIEDataRef::create(CASBuilder, Children, Data);
+    auto Ref = DIEDataRef::create(CASBuilder, Children, Data);
+    Data.clear();
```

Comment:
What does the Data.clear() do? Is that obvious for someone familiar with the code or should it be documented?